shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rule/TableRuleTest.java

```diff
@@ -154,4 +157,29 @@ public void assertActualDataNodesNotConfigured() {
         shardingTableRuleConfig.setTableShardingStrategy(new StandardShardingStrategyConfiguration("shardingColumn", "INLINE"));
         new TableRule(shardingTableRuleConfig, Arrays.asList("ds0", "ds1"), null);
     }
+
+    @Test
+    public void assertDatNodeGroups() {
+        LinkedList<String> dataSourceNames = new LinkedList<>();
+        String logicTableName = "table_0";
+        dataSourceNames.add("ds0");
+        dataSourceNames.add("ds1");
+        TableRule actual = new TableRule(dataSourceNames, logicTableName);
+        Map<String, List<DataNode>> dataNodeGroups = actual.getDataNodeGroups();
+        assertThat(dataNodeGroups.size(), is(2));
+        assertTrue(dataNodeGroups.get("ds0").contains(new DataNode("ds0", "table_0")));
+        assertTrue(dataNodeGroups.get("ds1").contains(new DataNode("ds1", "table_0")));
+    }
+
+    @Test
+    public void assertCreateAutoTableRuleWithdataSourceNames() {
```

Comment:
@yangsen00 This test method seems not cover auto table scenarios.

Reply:
hello, I have not solve this question，but this PR was closed, it will be OK?

Reply:
> hello, I have not solve this question，but this PR was closed, is this OK?

@yangsen00 You can submit a new pr to cover this case, I will reopen this issue.

Reply:
thanks. I will fix this today.

Reply:
I realized that I had solved this problem. Please close this issues，thanks